week2/assignment2_3_2/includes/sensor_data.h

```diff
@@ -0,0 +1,42 @@
+#ifndef SENSOR_DATA_H
+#define SENSOR_DATA_H
+
+
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+struct Sensor_reading
+{
+    int m_timestamp{0};
+    std::string m_sensor_id{};
+    float m_sensor_value{};
+
+    bool m_outlier_checked{ false };
+    bool m_is_outlier{ false };
+
```

Comment:
m_outlier_checked and m_is_outlier are breaking genarilisation of this object. They have only one purpose in one specific context.